ngraph/cmake/external_protobuf.cmake

```diff
@@ -100,6 +100,10 @@ else()
         set_target_properties(libprotobuf libprotobuf-lite PROPERTIES
             COMPILE_FLAGS "-Wno-unused-variable -Wno-inconsistent-missing-override")
     endif()
+
+    set_target_properties(libprotoc PROPERTIES
+        COMPILE_FLAGS "-Wno-unused-variable")
```

Comment:
I suppose this should be moved under the `if` above, because this does not work for MSVC compiler.

Reply:
that's right